Dockerfile

```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 ARG BUILD_SRC="/usr/local/src"
 
 ARG JANUS_VERSION="v1.0.0"
-ARG JANUS_LIBNICE_VERSION="0.1.17"
+ARG JANUS_LIBNICE_VERSION="0.1.18"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
```

Comment:
Are you sure this tag exists?
When checking https://github.com/alanxz/rabbitmq-c I see that v0.11.0 is the latest tag.

Reply:
I seems like there is an issue in the official documentation: https://janus.conf.meetecho.com/docs/DEPS.html

> rabbitmq-c: https://github.com/alanxz/rabbitmq-c (optional, v1.0.4, RabbitMQ)

I've fixed it already.